introduction_to_applying_machine_learning/identify_key_insights_from_textual_document/document_entity_recognition.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
```

Comment:
<p>Use present tense -&gt; it will send should be it send</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>